### PR TITLE
feat(wireless): add e2e tests for wireless configs

### PR DESCRIPTION
**Pull request by chethanbellechalu-intel**
Closes [AB#3262](https://dev.azure.com/rbheBoards/c7d9d701-a85f-45ff-91f6-26c56ff2c4de/_workitems/edit/3262)

### CODE REVIEW

File: cypress/integration/wireless/create.spec.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        cy.myIntercept("GET", "wirelessconfigs", {
+            statusCode: apiResponses.wirelessConfigs.getAll.success.code,
+            body: apiResponses.wirelessConfigs.getAll.success.response,
+        }).as("get-wireless2")
+
+        cy.get("button").contains("Add New").click()
+        cy.enterWirelessInfo(
+            wirlessFixtures.happyPath.profileName,
+            wirlessFixtures.happyPath.ssid,
+            "P@ssw0rd"
```
Comment (rsdmike) [inline cypress/integration/wireless/create.spec.js:33]: let's move this to the cypress env like the other passwords

File: cypress/integration/wireless/create-error.spec.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        cy.wait("@get-wireless3")
+
+        cy.get("button").contains("Add New").click()
+    })
+
+    it("invalid profile name", () => {
+        cy.enterWirelessInfo(
+          wirlessFixtures.wrong.profileName,
+          wirlessFixtures.happyPath.ssid,
+          "P@ssw0rd"
```
Comment (rsdmike) [inline cypress/integration/wireless/create-error.spec.js:31]: use cypress env for password

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: cypress/integration/wireless/create.spec.js (hunk 1 above)
Comment (chethanbellechalu-intel) [reply cypress/integration/wireless/create.spec.js]: Added new variable PSKPASSPHRASE to cypress.json

File: cypress/integration/wireless/create-error.spec.js (hunk 2 above)
Comment (chethanbellechalu-intel) [reply cypress/integration/wireless/create-error.spec.js]: Done

**Review by rsdmike — APPROVED** [review]

**Pull request merged**